xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -1228,6 +1228,9 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
   UpdateVideoTagField(parameterObject, "tag", tags, updatedDetails);
   details.SetTags(tags);
 
+  if (ParameterNotNull(parameterObject, "version"))
+    details.SetVersion(parameterObject["version"].asString());
+
```

Comment:
To add a parameter to the JSON API needs much more than just this. I would drop this from this PR and make part of a separate PR of JSON changes for this new feature. 

PR12863 adds some music params but should give you some hints what needs to change, or a look through these https://github.com/xbmc/xbmc/pulls?q=is%3Apr+label%3A%22JSON-RPC+API+Change%22+is%3Aclosed 
Ping me when you raise the PR and I will help you

Reply:
Thanks! I'll remove this from this PR.